android/src/main/java/sqip/flutter/SquareInAppPaymentsFlutterPlugin.java

```diff
@@ -1,5 +1,5 @@
 /*
- Copyright 2018 Square Inc.
+ Copyright 2022 Square Inc.
```

Comment:
nit, we don't have to change this header.

Reply:
Okay I have undo this.